package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "as-lunatic",
-  "version": "0.0.4",
-  "types": "assembly/index.d.ts",
+  "version": "0.0.1",
```

Comment:
Should we bump the version to `0.0.5`?

Reply:
This is a major change in the way the library works. I would recommend bumping to 0.1.0 in this package.